### ISSUE TITLE
Not possible to make a delta time tag now

### ISSUE THREAD

**Issue by crucialfelix** [russellmcc/node-osc-min#15]
It looks like you decided not to export deltaTimetag saying that it is easy for anybody to implement the function themselves.

But since you also haven't exported timestampToTimetag it means that we can't actually do that.

There is no way to create a time tag now except by using a Date object, and that is limited to millisecond accuracy.

Please add the function back in and/or export timestampToTimetag as well.

The code is already in the node-osc-min its just whether you expose it at the top level or not.

"minimal" to me means that it doesn't include socket libraries, servers etc. It just does conversion of bundles and messages to Buffers

**Comment by crucialfelix** [comment]
I have to reimplement all of this in order to get a deltaTimetag function:

```
UNIX_EPOCH = 2208988800
TWO_POW_32 = 4294967296

exports.deltaTimetag = (seconds, now) ->
  n = (now ? new Date()) / 1000
  return exports.timestampToTimetag(n + seconds)

# Convert a unix timestamp (seconds since jan 1 1970 UTC)
# to NTP timestamp array
exports.timestampToTimetag = (secs) ->
  wholeSecs = Math.floor(secs)
  fracSeconds = secs - wholeSecs
  return makeTimetag(wholeSecs, fracSeconds)

makeTimetag = (unixseconds, fracSeconds) ->
  # NTP epoch is 1900, JavaScript Date is unix 1970
  ntpSecs = unixseconds + UNIX_EPOCH
  ntpFracs = Math.round(TWO_POW_32 * fracSeconds)
  return [ntpSecs, ntpFracs]
```

**Comment by russellmcc** [comment]
Ah, yes you definitely should be able to create a high accuracy time stamp.  Correct me if I'm wrong, but couldn't you just pass in a UNIX time stamp into `toBuffer`?  Is there some reason you need to use the timetag outside of osc-min?

Definitely any common task should be handled by this library, but there is a risk of api creep, so I don't want to add any unnecessary function.  Definitely I'm okay with exposing `timestampToTimetag` if it's useful.

**Comment by crucialfelix** [comment]
> Is there some reason you need to use the timetag outside of osc-min?

Yes, I'm trying to make a timetag with full timing resolution.  

This is a library of OSC utilities, there is no reason not to expose this function.

**Comment by russellmcc** [comment]
Makes sense!  I'll expose it tonight.  I'll probably add a `timetagToTimestamp` so the app matches the one for Dates.  Does that make sense?

**Comment by crucialfelix** [comment]
It would be symmetrical, so not a bad idea.

thanks !

**Comment by russellmcc** [comment]
Fixed in 66fd4e18
